importer/Uploader.py

```diff
@@ -183,7 +183,7 @@ def make_pywikibot_item(self, value, prop=None):
         elif prop == PROPS["commonscat"] and utils.commonscat_exists(value):
             val_item = value
         else:
-            val_item = value
+            val_item = value.strip()
```

Comment:
Also add the strip() to L114 of Monuments.py (to make preview output the same as Uploader would)

Reply:
Done.